### PR TITLE
review dashboard (no more table positioning)

### PR DESCRIPTION
**Pull request by orthagh**

### CODE REVIEW

**Comment by trasher** [comment]
With Firefox:
- before
board_orig-fullpage
- after
board_rewrite-fullpage

**Comment by orthagh** [comment]
Fixed

**Comment by trasher** [comment]
I still have the issue with your latest commits and browser cache cleaned :/

File: css/views.css [hunk 1]
```diff
@@ -838,11 +838,51 @@ table.runs td.comment {
...
 
 .copyleft {
   display:inline-block;
   transform: rotate(180deg);
-}
\ No newline at end of file
+}
+
+.fi_board {
+   width: 1300px;
```
Comment (trasher) [inline css/views.css]: Why this value? It's not compliant with "small" screens and its breaks responsiveness, does'nt it?

**Pull request merged**